packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.test.tsx

```diff
@@ -244,25 +248,20 @@ describe('CompleteResetPassword page', () => {
   });
 
   describe('account has recovery key', () => {
-    account = {
+    const accountWithRecoveryKey = {
       resetPasswordStatus: jest.fn().mockResolvedValue(true),
       completeResetPassword: jest.fn().mockResolvedValue(true),
       hasRecoveryKey: jest.fn().mockResolvedValue(true),
     } as unknown as Account;
 
     it('redirects as expected', async () => {
-      account = {
-        resetPasswordStatus: jest.fn().mockResolvedValue(true),
-        completeResetPassword: jest.fn().mockResolvedValue(true),
-        hasRecoveryKey: jest.fn().mockResolvedValue(true),
-      } as unknown as Account;
+      lostRecoveryKey = false;
+      renderSubject(accountWithRecoveryKey);
 
-      renderSubject(account);
-
-      screen.getByLabelText('Loading…');
```

Comment:
I think we should have a loading spinner check _somewhere_ at least before the token check call.